tests/test_adjoint_jacobian.py

```diff
@@ -214,25 +252,29 @@ def test_Rot_gradient(self, stateprep, theta, dev):
 
         tape.trainable_params = {1, 2, 3}
 
-        calculated_val = dev.adjoint_jacobian(tape)
+        method = dev.compute_derivatives if dev._new_API else dev.adjoint_jacobian
+        calculated_val = method(tape)
```

Comment:
This block repeats a lot, maybe it makes sense to have it as a function. This kind of improvement always helps development and updates to tests.